sdk/storage/azure-storage-common/src/main/java/com/azure/storage/common/implementation/AccountSasImplUtil.java

```diff
@@ -57,7 +57,7 @@ public AccountSasImplUtil(AccountSasSignatureValues sasValues, String encryption
         this.sasIpRange = sasValues.getSasIpRange();
         this.services = sasValues.getServices();
         this.resourceTypes = sasValues.getResourceTypes();
-        this.encryptionScope = encryptionScope;
+        this.encryptionScope = encryptionScope == null ? sasValues.getEncryptionScope() : encryptionScope;
```

Comment:
TODO (Sean/Rabab): We need to revisit.  I believe we implemented Blob Encryption Scope SAS with STG 78, and we may not have patterned matched Encryption Scope SAS correctly in Data Lake.